src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,112 @@
+$side-margins-in-card: 16px;
```

Comment:
this `margins` you use, they should be a paddings for your CARD, not the elements inside